kubeflow/fairing/builders/podman/podman.py

```diff
@@ -0,0 +1,83 @@
+import json
+import logging
+
+import os
+#from podman import Client
+
+from kubeflow.fairing.builders.base_builder import BaseBuilder
+from kubeflow.fairing.builders import dockerfile
+from kubeflow.fairing.constants import constants
+
+logger = logging.getLogger(__name__)
+
+
+class PodmanBuilder(BaseBuilder):
+    """A builder using the local Podman"""
+
+    def __init__(self,
+                 registry=None,
+                 image_name=constants.DEFAULT_IMAGE_NAME,
+                 base_image=constants.DEFAULT_BASE_IMAGE,
+                 preprocessor=None,
+                 push=True,
+                 dockerfile_path=None,
+                 tls_verify=False):
+        super().__init__(
+            registry=registry,
+            image_name=image_name,
+            push=push,
+            base_image=base_image,
+            preprocessor=preprocessor,
+            dockerfile_path=dockerfile_path)
+        self.tls_verify = tls_verify
```

Comment:
Would you please add defailed description for the parameters? the format can be as below, so that we can update the Fairing API docs for the builder later. please add docString for all function and Classes, Thanks.
```
        ""Description for the function....

        :param registry: basic description ...
        :param image_name: basic description ...

    
        """
```